test/opsem2/bswap/bswap-ntohs-inline-asm_sat.c

```diff
@@ -0,0 +1,20 @@
+//; RUN: %sea "%s" 2>&1 | OutputCheck %s
+// CHECK: ^Error: assertion failed
+// CHECK: ^sat$
+
+#include "seahorn/seahorn.h"
+#include <arpa/inet.h>
+#include <stdio.h>
+
+extern uint16_t nd_uint16_t(void);
+
+int main(int argc, char **argv) {
+  uint16_t a = nd_uint16_t();
+  uint16_t b = nd_uint16_t();
+  __VERIFIER_assume(a == b);
```

Comment:
we can use assume(...). : https://sourcegraph.com/github.com/seahorn/seahorn@dev10/-/blob/include/seahorn/seahorn.h#L39